src/db.h

```diff
@@ -141,6 +141,17 @@ enum data_kind {
 const char *
 db_data_kind_label(enum data_kind data_kind);
 
+
+/* Indicates user marked status on a track */
+enum flag_kind {
+  FLAG_KIND_NA = 0,     // unset
+  FLAG_KIND_DELETE,
+  FLAG_KIND_REXCODE,
+  FLAG_KIND_REVIEW,
+
+  FLAG_KIND_MAX         // unused - keep last in enum
```

Comment:
Drop this, inconsistent with other enums and probably not necessary (especially if using bit field enum). Also, shouldn't it actually be allowed for the user to mark with values outside the enum range? I suppose we cannot foresee all the uses of this field.